frontend/src/components/NavigationBar.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import ReactDOM from 'react-dom';
+import NavigationBar from './NavigationBar';
+
+describe('Navigation Bar', () => {
+    it('renders without crashing', () => {
+        const navWrapper = shallow(<NavigationBar/>);
+        expect(navWrapper).toBeTruthy()
+     });
```

Comment:
It's good to test the basic functionality of the app - however, is "renders without crashing" the best description of the action being performed here?  Consider rewording to describe what the app is doing here.